### PR TITLE
[1] release/STYLE-12959 updates project

### PR DESCRIPTION
**Pull request by Figedi**
## 🤔 Rules for reviewers
- Reject large PRs if it lacks detailed description / story behind it
- Large PRs may be done pairing with author
- "Approve" only if you approve of functional impact
- "Comment" only if you just did a code-style review
- Reject PRs with insufficient test cases
- **Meaningful rejections make us better**

## 📖 Description
This PR:
- updates the project to the latest standards & dep-versions

## 🔍 Review
- No thorough review required, follow-up PRs for the actual refactoring will follow

## 💥 Risks 
Low, this just updates things

## 🚀 Impact 
Low, this just updates things

## 👁‍🗨 Unknowns
None

 ## 🎶 Notes
None

### CODE REVIEW

**Comment by ansgarS** [comment]
Reject large PRs :D

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+version: 2.1
```
Comment (samnajian) [inline .circleci/config.yml]: This is a public repo! we can't have a CI config, can we?

Comment (Figedi) [reply .circleci/config.yml]: all ci-orbs are public and there are no secrets in that orb

Comment (samnajian) [reply .circleci/config.yml]: but doesn't a push from public users create a build in out pipeline?

Comment (Figedi) [reply .circleci/config.yml]: yes ofc, that is how open source works 🤔

**Pull request merged**